### PR TITLE
Set LANG env variable

### PR DESCRIPTION
**Pull request by alangregory**
Fix to issue #42

### CODE REVIEW

**Comment by mhor** [comment]
@alangregory thank you!!! I was really busy last weeks.

**Comment by mhor** [comment]
released in [1.5.2](https://github.com/mhor/php-mediainfo/releases/tag/1.5.2)

- fix #42

**Comment by mhor** [comment]
@alangregory Ooops, I merge this to fast... Travis-ci build is currently break, could you fix it?

**Comment by alangregory** [comment]
I'll fix ENV on #44 in a moment.

File: src/Runner/MediaInfoCommandRunner.php [hunk 1]
```diff
@@ -63,10 +62,10 @@ public function __construct($filePath, array $arguments = null, $processBuilder
      */
     public function run()
     {
-        $this->processBuilder->add($this->filePath);
+        $lc_ctype = setlocale(LC_CTYPE, 0);
+        $this->processBuilder->add($this->filePath)->addEnvironmentVariables(array('LANG' => $lc_ctype));
```
Comment (mhor) [inline src/Runner/MediaInfoCommandRunner.php]: @alangregory could you replace ```addEnvironmentVariables(array('LANG' => $lc_ctype)``` by ```setEnv('LANG', $lc_type)``` https://github.com/symfony/symfony/blob/master/src/Symfony/Component/Process/ProcessBuilder.php#L142

Comment (alangregory) [inline src/Runner/MediaInfoCommandRunner.php]: Yes 


Atenciosamente, 




	

Alan Gregory 

Opcão Telecom 

Departamento de Redes 

+55 45 3253-2277 | 45 9907-9669 

alan @opcaonet.com.br 

skype: alan@opcaonet.com.br 



De: "Maxime Horcholle" <notifications@github.com> 
Para: "mhor/php-mediainfo" <php-mediainfo@noreply.github.com> 
Cc: "alan" <alan@opcaonet.com.br> 
Enviadas: Segunda-feira, 21 de dezembro de 2015 14:19:58 
Assunto: Re: [php-mediainfo] Set LANG env variable (#43) 



In src/Runner/MediaInfoCommandRunner.php : 
> @@ -63,10 +62,10 @@ public function __construct($filePath, array $arguments = null, $processBuilder
>       */
>      public function run()
>      {
> -        $this->processBuilder->add($this->filePath);
> +        $lc_ctype = setlocale(LC_CTYPE, 0);
> +        $this->processBuilder->add($this->filePath)->addEnvironmentVariables(array('LANG' => $lc_ctype)); 


@alangregory could you replace addEnvironmentVariables(array('LANG' => $lc_ctype) by setEnv('LANG', $lc_type) https://github.com/symfony/symfony/blob/master/src/Symfony/Component/Process/ProcessBuilder.php#L142

**Pull request merged**